modules/creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,195 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-with-aws-kms-key"]
+= Creating a ROSA cluster using custom AWS KMS key
+
+You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that will be used to encrypt both the root volumes of nodes as well as persistent volumes for mounted EBS claims.
+
+.Procedure
+
+. Create a custom AWS customer-managed KMS key:
++
+[source,terminal]
+----
+$ KMS_ARN=$(aws kms create-key --region $AWS_REGION --description 'Custom ROSA Encryption Key' --tags TagKey=red-hat,TagValue=true --query KeyMetadata.Arn --output text)
+----
++
+This command will save the ARN output of this custom key for further steps.
```

Comment:
```suggestion
This command saves the ARN output of this custom key for further steps.
```

Reply:
Do we need to spell out ARN?

Reply:
Replaced with "Amazon Resource Name (ARN)"